src/components/ProductList.tsx

```diff
@@ -83,29 +83,50 @@ const ProductList = () => {
     setSelectedCategory(category);
   };
 
+  // Modify last word 
```

Comment:
Unnecessary code comment.

Consider the same remark in other places.

See: [Necessary Comments - Clean Coder](https://blog.cleancoder.com/uncle-bob/2017/02/23/NecessaryComments.html)